tests/console/sysstat.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test sysstat basic functionalities
+# Maintainer: Sergio Rafael Lemke <slemke@suse.cz>
+
+use base 'consoletest';
+use utils 'zypper_call', 'systemctl';
```

Comment:
Please, replace with `qw(zypper_call systemctl)`, which is more widely accepted way for imports.